Expensify/App
# Context Menu rendering

### Details


### Fixed Issues

Fixes #2967

### Tests | QA Steps (Web | desktop)
1. Open Any conversation on E.cash.
2. Open the Context menu on any message.
3. Now try to open the context menu for another message. The previous message should close and a new one should open.
4. Open the context menu on any message.
5. Click somewhere outside the context menu. A context menu should close.

### Tests | QA Steps (Mobile | Mweb)

1. open Any conversation on E.cash
2. Hold press any message to open the context menu.
3. Click in the empty area should close the menu.
3. There should be no apparent changes and the context menu should behave as before.

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/121562741-5ef41a80-ca37-11eb-98f3-4b6cd684601e.mp4


#### Mobile Web


https://user-images.githubusercontent.com/24370807/121562414-0c1a6300-ca37-11eb-980a-36ad53d42ffd.mp4


#### Desktop

https://user-images.githubusercontent.com/24370807/121561838-7e3e7800-ca36-11eb-978e-ff15e66fb7e5.mp4
#### iOS


#### Android


https://user-images.githubusercontent.com/24370807/121562428-0f155380-ca37-11eb-9209-7f27ecb8095c.mp4
